src/WikiService.Models.Db/DbArticle.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.WikiService.Models.Db
+{
+  public class DbArticle
+  {
+    public const string TableName = "Articles";
+
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public string Content { get; set; }
+    public bool IsActive { get; set; }
+    public Guid CreatedBy { get; set; }
+    public DateTime CreatedAtUtc { get; set; }
+    public Guid? ModifiedBy { get; set; }
+    public DateTime? ModifiedAtUtc { get; set; }
+
+    public DbRubric Rubric { get; set; } 
```

Comment:
need Guid RubricId prop

Reply:
Sure! you're great! thanks!
Added to DbArticles and migration